### ISSUE TITLE
TypeError: object list can't be used in 'await' expression

### ISSUE THREAD

**Issue by Scathachnyx** [DIGITALCRIMINALS/OnlyFans#62]
I have 0 experience with this, but after a lot of time I've managed to do all steps, get the cookies and user-agent, it logs to my account and I can see all the profiles I follow, I chose one and then proceed to chose to download images, videos, etc. Then I get this:

Scrape Processing
Name: uuchan
Scrape: 0 = All | 1 = Images | 2 = Videos | 3 = Audios | 4 = Texts
2
Type: Profile
Type: Stories
No Stories Found.
Type: Posts
Scrape Attempt: 1/100
Type: Archived Posts
Processing Scraped Posts
  0%|                                                                                            | 0/1 [00:00<?, ?it/s]Traceback (most recent call last):
  File "C:\Users\Tiamat\Desktop\OnlyFans\of-datascrapper\start_ofd.py", line 60, in <module>
    loop.run_until_complete(main())
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2032.0_x64__qbz5n2kfra8p0\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\Users\Tiamat\Desktop\OnlyFans\of-datascrapper\start_ofd.py", line 44, in main
    api = await main_datascraper.start_datascraper(
  File "C:\Users\Tiamat\Desktop\OnlyFans\of-datascrapper\datascraper\main_datascraper.py", line 93, in start_datascraper
    await main_helper.process_names(
  File "C:\Users\Tiamat\Desktop\OnlyFans\of-datascrapper\helpers\main_helper.py", line 939, in process_names
    result = await module.start_datascraper(authed, username, site_name)
  File "C:\Users\Tiamat\Desktop\OnlyFans\of-datascrapper\modules\onlyfans.py", line 160, in start_datascraper
    await prepare_scraper(authed, site_name, item)
  File "C:\Users\Tiamat\Desktop\OnlyFans\of-datascrapper\modules\onlyfans.py", line 823, in prepare_scraper
    unrefined_set = await tqdm.gather(tasks, **settings)
  File "C:\Users\Tiamat\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\tqdm\asyncio.py", line 79, in gather
    res = [await f for f in cls.as_completed(ifs, loop=loop, timeout=timeout,
  File "C:\Users\Tiamat\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\tqdm\asyncio.py", line 79, in <listcomp>
    res = [await f for f in cls.as_completed(ifs, loop=loop, timeout=timeout,
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2032.0_x64__qbz5n2kfra8p0\lib\asyncio\tasks.py", line 614, in _wait_for_one
    return f.result()  # May raise f.exception().
  File "C:\Users\Tiamat\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\tqdm\asyncio.py", line 76, in wrap_awaitable
    return i, await f
TypeError: object list can't be used in 'await' expression

**Comment by americanseeder1865** [comment]
I wonder if your global python environment is polluted for whatever reason and you are in a state of dependency hell. 

Can you run the scraper in [a virtual enviroment](https://docs.python.org/3/tutorial/venv.html) and see if anything changes?

**Comment by Scathachnyx** [comment]
Thanks a lot for the answer! I finally made it work. 

First I tried the virtual enviroment, used the command to create it but really didn't know what to do after that as like I said I have no idea about programming languages and how all of this works.

Then I tried the C++ stuff you mentioned, on "Visual Studio Build Tools 2019" I had installed the stuff mentioned on post #43 but I did a 'C++' search and one called C++/CLI with building tools compatibility v142' got my attention as one of the requisites I had installed was 'MSVC cv142 building tools". Not sure if it worked because of this, but the fact is after I installed it everything worked, the auth.json and other folders were created and I succefully downloaded all files from an OnlyFans. 

So thanks again for that~
